FantasyUniverse/Views/App views/QuestionLabelView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  QuestionLabelView.swift
+//  FantasyUniverse
+//
+//  Created by TarasPeregrinus on 04.02.2022.
+//
+
+import SwiftUI
+
+struct QuestionLabelView: View {
+    let text: String?
```

Comment:
Why not to require a non-optional string? Does this view even make sense without this text?